src/routes/settings-root-proxy-route/settings-root-proxy-route.js

```diff
@@ -34,7 +34,7 @@ const SettingsRootProxyRoute = ({
   useEffect(() => {
     getProxyTypes(match.params.kbId);
     getRootProxy(match.params.kbId);
-  }, []);
+  }, [getProxyTypes, getRootProxy, match.params.kbId]);
```

Comment:
Please make sure there aren't duplicate requests

Reply:
I see that there are no duplicate requests.